setup/resources/user-workloads.yaml

```diff
@@ -0,0 +1,157 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: nodejs-sample
+objects:
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      namespace: ${NAMESPACE}
```

Comment:
What happens if someone adds a new resource but forgets to set `namespace: ${NAMESPACE}`? Will the resource be created in an unexpected namespace?

Reply:
I need to check that. I wanted to enforce the namespace on the template objects between the `Process` and `Apply` funcs, but I couldn't find an easy way to do it. But I guess I should give it another try, so we don't have to deal with this parameter at all...

Reply:
done in 00c8f2e: there is no template parameter anymore, yet the objects' namespace is enforced just before creation.